src/Cech_complex/include/gudhi/Cech_complex_blocker.h

```diff
@@ -73,6 +75,8 @@ class Cech_blocker {
     CGAL::NT_converter<FT, Filtration_value> cast_to_fv;
     Filtration_value radius = 0;
     bool is_min_enclos_ball = false;
+    Point_cloud points;
+    points.reserve(boost::size(sc_ptr_->simplex_vertex_range(sh)));
```

Comment:
`sc_ptr_->dimension(sh)+1` should give the same answer more naturally than counting the range.

Reply:
Indeed, I was looking for something like this, thanks!